common/lib/xmodule/xmodule/tests/__init__.py

```diff
@@ -59,6 +61,8 @@ def __init__(self, **kwargs):
         kwargs.setdefault('id_reader', id_manager)
         kwargs.setdefault('id_generator', id_manager)
         kwargs.setdefault('services', {}).setdefault('field-data', DictFieldData({}))
+        self.tracker = DjangoTracker()
+        tracker.register_tracker(self.tracker)
```

Comment:
I've mocked emit here on init function. I've used this decorator: 
@mock.patch('eventtracking.tracker.emit') 
is that a proper way to do this? I believe there is no need to delete these lines if emit function is mocked, correct? (nothing should be logged)